releasenotes/notes/low-memory-alert-units-a6fde380ff9b7839.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Adds the word GiB to the low memory alert to make more sense of the alert.
```

Comment:
I would say more simply:

Add unit to LowMemory alert description.

Reply:
Hopefully should be resolved now 😄